packages/api-docs/src/generatePageJsonFiles.ts

```diff
@@ -1,7 +1,7 @@
 import * as path from 'path';
 import { generateJson } from './PageJsonGenerator';
 
-// Generate JSON for office-ui-fabric-react, styling, utilities, and merge-styles
+// Generate JSON for office-ui-fabric-react, styling, utilities, merge-styles, and legacy
```

Comment:
@natalieethell This seems to work okay but I'd appreciate if you could take a look